playbooks/common-dev/vars/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+release: "{{ ansible_distribution_version.split('.')[0] }}"
+server_group: "server-{{ inventory_hostname }}"
+server: "{{ groups[server_group][0] }}"
+version_map:
+  'nightly': 'nightly'
+  '3.0': '1.11'
+katello_version: "nightly"
+foreman_version: "nightly"
```

Comment:
You could instead define this in `common/defaults/main.yml` and then override them:

```
---
 - hosts: server capsule
   roles:
     - {role: common, katello_version: 'nightly', foreman_version: 'nightly'}
     - etc_hosts
     - capsule-dev
```